<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="descriptorIndexStartsAt0Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Descriptor Index Starts at 0 (Coming Soon)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="descriptorIndexStartsAt0CheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="primarilyPreferredLocaleLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Primarily Preferred Locale (Coming Soon)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="primarilyPreferredLocaleComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="secondarilyPreferredLocaleLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Secondarily Preferred Locale (Coming Soon)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="secondarilyPreferredLocaleComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="automaticPNGDownloadLabel">
        <property name="text">
         <string>Automatic PNG Download</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>16</number>
        </property>
        <item>
         <widget class="QCheckBox" name="automaticPNGDownloadCheckBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearLogoCacheButton">
          <property name="text">
           <string>Clear Cache</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="transposeConnectionMatrixLabel">
        <property name="text">
         <string>Show Talkers Vertically</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="transposeConnectionMatrixCheckBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="automaticCheckForUpdatesLabel">
        <property name="text">
         <string>Automatic Check For Updates</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="automaticCheckForUpdatesCheckBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="checkForBetaVersionsLabel">
        <property name="text">
         <string>Check For Beta Versions</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkForBetaVersionsCheckBox"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="themeColorLabel">
        <property name="text">
         <string>Theme Color</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="themeColorComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>AVDECC Controller</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="enableAEMCacheLabel">
        <property name="text">
         <string>Enable AEM Cache</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="enableAEMCacheCheckBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="enableAdvertisingLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Enable Advertising (Coming Soon)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="enableAdvertisingCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="controllerIDLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="controllerIDLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Controller ID (Coming Soon)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
